PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/PBMCore/MediaData.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  PBMMediaDataTTT.swift
+//  PrebidMobileRendering
+//
+//  Copyright © 2021 Prebid. All rights reserved.
+//
+
+import Foundation
+
+@objc(PBMMediaData) public class MediaData: NSObject {
```

Comment:
The same questions about `@objc`. Need to develop the correct way from the very begining.